docs/runtimes/micronaut.md

```diff
@@ -0,0 +1,160 @@
+   git push
+   ```
+7. Create a new branch called `env-dev`, and push it to trigger a build:
+   ```bash
+   git checkout -b env-dev
+   git push -u origin env-dev
+   ```
+8. Go to your GitHub project, and check that the GitHub Action is running.
+9. You can go to the [Azure Portal](https://aka.ms/nubesgen-portal) to check the created resources.
+10. Once deployed, you can access the application at `https://app-demo-XXXX-XXXX-XXXX-XXXX-dev-001.azurewebsites.net/`.
+   Add a TODO to the database:
+   ```bash
+   > curl -X POST 'https://app-demo-XXXX-XXXX-XXXX-XXXX-dev-001.azurewebsites.net/graphql' \
+          -H 'content-type: application/json' \
+          --data-binary '{"query":"mutation { createToDo(title:\"Create GraphQL Guide\", author:\"Tim Yates\") { id } }"}'
+
+   {"data":{"createToDo":{"id":"1"}}}
+   ```
+   List the current TODOs:
+   ```bash
+   > curl -X POST 'https://app-demo-XXXX-XXXX-XXXX-XXXX-dev-001.azurewebsites.net/graphql' \
+          -H 'content-type: application/json' \
+          --data-binary '{"query":"{ toDos { title, completed, author { username } } }"}'
+
+   {"data":{"toDos":[{"title":"Create GraphQL Guide","completed":false,"author":{"username":"Tim Yates"}}]}}
+   ```
+10. Once you have finished, you should clean up your resources:
+   1. Delete the resource group that was created by NubesGen to host your resources, which is named `rg-demo-XXXX-XXXX-XXXX-XXXX-001`.
+   2. Delete the storage account used to store your Terraform state, in the `rg-terraform-001` resource group.
+
+## Which Azure resources will be created
+
+If you deploy your Micronaut application to an Azure App Service instance, NubesGen will generate:
+
+- An [Azure App Service plan](https://aka.ms/nubesgen-app-service-plans) to define the type of App Service instance you will use.
+- An [Azure App Service instance](https://aka.ms/nubesgen-app-service), configured to run Java code natively.
+
+If you deploy your Micronaut application to an Azure Function, NubesGen will generate:
+
+- An [Azure App Service plan](https://aka.ms/nubesgen-app-service-plans) to define the type of Azure Functions instance you will use.
+- An [Azure Functions instance](https://aka.ms/nubesgen-functions), configured to run Java code natively.
+- An [Azure Storage Account](https://aka.ms/nubesgen-storage), to store your Java application.
+
+## Maven vs Gradle
+
+NubesGen supports both Maven and Gradle, so you can use the build system you prefer.
+For Maven, change `runtime=micronaut_gradle` to `runtime=micronaut` in Step 5 above.
+
+## Configuration options
+
+In the generated `terraform/modules/app-service/main.tf` file, NubesGen will configure some environment variables for your application.
+Those are standard Micronaut properties, so your Micronaut application should be automatically configured (for example: your database connection should be working out-of-the-box).
+
+- `DATASOURCES_DEFAULT_URL`: the JDBC URL to your database
```

Comment:
I'm surprised, don't you have standardised properties like Spring Boot and Quarkus?

Reply:
Oh sorry, the standardised properties seem to be those ones!

Reply:
Yeah 👍